### PR TITLE
fix #5525 too many Nones in CondaHttpError

### LINKED ISSUE #5525: Fix all the `None`s in CondaHTTPError
```
CondaHTTPError: HTTP None None for url https://repo.continuum.io/pkgs/free/win-64/repodata.json.bz2
Elapsed: None
```

### PR DESCRIPTION
**Pull request by kalefranz**

### CODE REVIEW

File: conda/exceptions.py [hunk 1]
```diff
@@ -355,6 +355,10 @@ def __init__(self, message, url, status_code, reason, elapsed_time, response=Non
         _message += "CF-RAY: %s\n\n" % cf_ray if cf_ray else "\n"
         message = _message + message
 
+        status_code = status_code or '000'
```
Comment (goanpeca) [inline conda/exceptions.py]: Great, although `000` might be confusing?

Comment (kalefranz) [inline conda/exceptions.py]: Would `---` be better?

Comment (goanpeca) [inline conda/exceptions.py]: Maybe, cause 000 is not really a defined valid code

**Comment by kalefranz** [comment]
I actually think that's the point though... There's no HTTP response code to display.

**Comment by goanpeca** [comment]
Ok then go ahead :-p

**Comment by kalefranz** [comment]
Mind approving the PR? :)

**Pull request merged**